CHANGELOG.md

```diff
@@ -41,6 +41,10 @@
 * Fix #2699: Add support for `@EnableOpenShiftMockClient` for OpenShiftClient
 * Fix #2588: Support for Server side dry runs
 
+- CamelK extension
```

Comment:
Please reference the associated issue, using the format of the other changelog entries.

Reply:
https://github.com/ioforks/kubernetes-client/pull/4